### ISSUE TITLE
Consumption Reporting - Handling of access_network_changed events

### ISSUE THREAD

**Issue by dsilhavy** [5G-MAG/rt-5gms-media-session-handler#42]
## Feature description
TS 26.512 Section 4.7.4 defines procedures for consumption reporting. One requirement is the handling of access network changes:

> If the consumption reporting procedure is activated, the Media Session Handler shall produce and submit a consumption report to the 5GMSd AF when any of the following conditions occur: Upon determining an access network change (e.g., unicast to eMBMS, or vice versa), if the clientConsumptionReportingConfiguration.accessReporting property is set to True.

For that reason the MediaStreamHandler sends a `access_network_changed ` event once the access network has changed (see https://github.com/5G-MAG/rt-5gms-media-stream-handler/issues/56). The MediaSessionHandler then needs to check if `clientConsumptionReportingConfiguration.accessReporting` is set to `true`. If `true` the MediaSessionHandler requests a consumption report from the MediaStreamHandler similar to what is already being done once the `reportingInterval` has elapsed. 

Whenever a consumption report is produced, the Media Session Handler shall reset its reporting interval timer to the value of the clientConsumptionReportingConfiguration.reportingInterval property and it shall begin countdown of the timer again.

**Comment by shilinding** [comment]
If we detect the change of an access network directly in the MediaSessionHandler, dispatching and handling of _access_network_changed_ events are not needed, right?

**Comment by dsilhavy** [comment]
> If we detect the change of an access network directly in the MediaSessionHandler, dispatching and handling of _access_network_changed_ events are not needed, right?

Yes, let's continue discussion in https://github.com/5G-MAG/rt-5gms-media-stream-handler/issues/56 and then adjust the issue description

**Comment by dsilhavy** [comment]
Agreed to put this to the backlog as the desired behavior needs to be clarified.
